### ISSUE TITLE
Integrate Medusa with Sanity, hard to follow Step 6

### ISSUE THREAD

**Issue by ollu** [medusajs/medusa#10409]
### What Medusa version and documentation are you using?

v2

### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/medusajs/medusa/issues

### Issue Summary

I'm following this guide https://docs.medusajs.com/resources/integrations/guides/sanity and in the **Step 6** I find it hard to understand what to update the `sync.ts` file with.

The instructions says "Next, you want to sync the retrieved products. So, replace the TODO with the following:".

If I follow the instructions the file gets broken, since there is an import statement as well as some code logic that should be placed in different places.

### How can this issue be resolved?

1. Break code example into two sections to make it clear where each section should go
2.
3.
...

### Are you interested in working on this issue?

- [ ] I would like to fix this issue

**Comment by shahednasser** [comment]
Thank you for reporting this. you have a point i can see where the confusion comes from. Will look into improving that section 👍🏻
